package.json

```diff
@@ -51,29 +49,15 @@
   },
   "eslintConfig": {
     "extends": [
-      "kentcdodds",
-      "kentcdodds/ava"
-    ]
-  },
-  "nyc": {
-    "all": true,
-    "check-coverage": true,
-    "branches": 100,
-    "function": 100,
-    "lines": 100,
-    "statements": 100,
-    "reporter": [
-      "text",
-      "lcov"
-    ],
-    "include": [
-      "src"
+      "kentcdodds"
     ],
-    "sourceMap": false,
-    "instrument": false,
-    "require": [
-      "babel-register"
-    ]
+    "env": {
+      "jest": true
+    }
+  },
+  "jest": {
+    "cacheDirectory": "./node_modules/.cache",
```

Comment:
This is here so travis will hang onto the cache between runs (speeding up subsequent test runs)